tests/utils/test_modeling_utils.py

```diff
@@ -1919,21 +1918,23 @@ def test_loading_is_fast_on_gpu(self, model_id: str, max_loading_time: float):
             parser.add_argument("max_loading_time", type=float)
             args = parser.parse_args()
 
-            device = torch.device("cuda:0")
+            device_type = torch.accelerator.current_accelerator().type if hasattr(torch, "accelerator") else "cuda"
```

Comment:
maybe use `is_torch_accelerator_available` like what you did in bark?

Reply:
sure, done, thx